test/styles/RRR.R

```diff
@@ -39,38 +48,110 @@ fun_call(argument1,
 fun_call(
     argument1,
     argument2
-    )
+)
 
 ## 3
 fun_call(parameter = (
     stuff
-    ),
-         argument)
+),
+argument)
 
 ## 4
 fun_call(parameter = fun_argument(
-             stuff
-    ),
-         argument)
+    argument1
+),
+argument2)
+
+## 5
+fun_call(parameter = fun_argument(argument1,
+                                  argument2
+                                  )
+        ,
+         argument3)
```

Comment:
Well. Take this example. Right now (because `ess-offset-arguments` = nil, and `ess-offset-arguments-newline` = `(t)`) I get either this

```
fun_call(parameter = fun_argument(argument1,
                                   argument2
                                   )

```

or this

```

fun_call(parameter = fun_argument(
    argument1,
    argument2
 )
```

I cannot get anything in between because `outer-parameter` setting is ignored. 

So my concern is three fold. First, after `{` this makes sense, doe it really make sense after `(` given that it's a very common pattern and all subsequent arguments will be shifted to 0. Second, it's not the current behavior, and folks will complain. Third, is there a way to make it 

```

fun_call(parameter = fun_argument(
             argument1,
             argument2
         )
```

Without affecting the indentation after last `{`?